### ISSUE TITLE
A11y_SQLAzureDataStudio_Query_Create&Run_ScreenReader: Content present on the screen not read by screen reader.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9408]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark; #DesktopApp;#VoiceOver;#MacOs;#MAS1.3.1;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Mac OS:10.15.3
MAS References:MAS1.3.1
Screen reader: VoiceOver

**Repro Steps:**
1.Launch Azure Data Studio.
2.Navigate to File > New Query
3.Connect to a Server in the Query Editor and choose the credential, and optionally the database to connect to the server.
4.Type a SQL query statement.
5.Click Run.
6.Navigate to Messages section.
7.Check if content is properly read by screen reader.

**Actual:**
Content present on the screen below messages section is not read by Voice Over screen reader.

**Expected:**
All the content present on the screen should be read by the screen reader.

**User Impact:**
Screen reader users might receive less information of what is present on the screen.

**Attachment link reference:**
[9408_A11y_SQLAzureDataStudio_Query_Create&Run_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4291311/9408_A11y_SQLAzureDataStudio_Query_Create.Run_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
it seems the keyboard navigation has regressed for query results elements.

**Comment by manaswita-chougule** [comment]
Verified the issue in below environment :
Version: 1.41.1 (Universal)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:49:19.454Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.3.0

The issue is fixed. Hence adding closed tag. 
#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-SQLADS-MAC-Feb2020;
